components/BottomNavigation/examples/BottomNavigationTitleVisibilityChangeExample.swift

```diff
@@ -71,21 +72,26 @@ class BottomNavigationTitleVisibilityChangeExample: UIViewController, MDCBottomN
   }
   
   func addInstructionLabel() {
-    let instructionLabel = UILabel()
     instructionLabel.numberOfLines = 0
+    instructionLabel.textAlignment = .center
     instructionLabel.lineBreakMode = .byWordWrapping
     instructionLabel.text = "Choose the Home tab to make all titles disappear, and any other tab to make them reappear."
     view.addSubview(instructionLabel)
-    let size = instructionLabel.sizeThatFits(view.bounds.size)
-    let instructionFrame = CGRect(x: 0,
-                                  y: view.bounds.height / 2 - size.height / 2,
-                                  width: size.width,
-                                  height: size.height)
-    instructionLabel.frame = instructionFrame
   }
   
   override func viewWillLayoutSubviews() {
     super.viewWillLayoutSubviews()
+
+    var viewBounds = view.bounds;
+    if #available(iOS 11.0, *) {
+      viewBounds = UIEdgeInsetsInsetRect(viewBounds, view.safeAreaInsets)
+    }
+    let labelWidth = min(viewBounds.size.width - 32, 480);
```

Comment:
Magic numbers... -32, 480 - What do these mean?

Reply:
32 is 16 point margins on either side if the label spans the full width of the view. I believe this matches [Material.io guidelines for margins](https://material.io/design/layout/responsive-layout-grid.html#columns-gutters-margins).

480 looked pretty nice when the layout was very wide.  If you have a reason to pick a different max width, I'd be happy to use it. :)